locales/en.json

```diff
@@ -49,8 +49,8 @@
 	"Help": "Help",
 	"Upload Image": "Upload Image",
 	"Menu": "Menu",
-	"This page need refresh": "This page need refresh",
-	"You have an incompatible client version.": "You have an incompatible client version.",
+	"This page need refresh": "This page needs a refresh",
```

Comment:
It can't be. The only case where you can omit a determiner ("a" or "the") is with a pronoun. "needs a" is required gere.